parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -184,6 +184,7 @@ ensureMonitorService
 # https://github.com/kubernetes/kubernetes/issues/51017
 # can remove when fixed
 cleanupRetaggedImages
```

Comment:
remove before merge, just for testing runtime retag

Reply:
just curious, how did you test this? did you check the tags in a node in moon cake, in a standalone environment?

Reply:
i changed the logic a bit, I testing by forcing cleanup and then running retag in public cloud, then changing the if condition from != back to ==